docs/howto/add_pagination_links.md

```diff
@@ -0,0 +1,46 @@
+# How to add pagination links
+
+Pagination links will be included in your response automatically as long as the resource is paginated and if you are using a ```JSON-API``` adapter. The others adapters does not have this feature.
+
+If you want pagination links in your response, use [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate).
+
+```ruby
+#kaminari example
+@posts = Kaminari.paginate_array(Post.all).page(3).per(1)
+render json: @posts
+
+#will_paginate example
+@posts = Post.all.paginate(page: 3, per_page: 1)
+render json: @posts
+```
```

Comment:
Hey, @bf4 tks for the review.

I believe that, using `WillPaginate::Collection` we are overcomplicated it, since `will_paginate` already do it for us.
http://www.rubydoc.info/github/mislav/will_paginate/WillPaginate/Collection
>```
Array.class_eval do
  def paginate(page = 1, per_page = 15)
    WillPaginate::Collection.create(page, per_page, size) do |pager|
      pager.replace self[pager.offset, pager.per_page].to_a
    end
  end
end
```

What do you think doing this way?

###### Kaminari examples
```ruby
#array
@posts = Kaminari.paginate_array([1, 2, 3]).page(3).per(1)
render json: @posts

#active_record
@posts = Post.page(3).per(1)
render json: @posts
```

###### WillPaginate examples

```ruby
#array
@posts = [1,2,3].paginate(page: 3, per_page: 1)
render json: @posts

#active_record
@posts = Post.page(3).per_page(1)
render json: @posts
```